pyvista/plotting/widgets.py

```diff
@@ -549,6 +556,9 @@ def _the_callback(widget, _event):
             if not normal_rotation:
                 plane_widget.GetNormalProperty().SetOpacity(0)
 
+            if not outline_visible:
+                plane_widget.GetOutlineProperty().SetOpacity(0)
```

Comment:
This method accepts a double between 0 and 1.  Should we instead make this a `outline_opacity`?  We could also consider adding in a dict for any property to be set.

Reply:
resolved in 40e30759499e13c3658a6d465b831f6114724c8f